pandas/tests/arrays/test_integer.py

```diff
@@ -771,6 +771,17 @@ def test_integer_array_constructor_copy():
     assert result._mask is not mask
 
 
+def test_integer_Series_iter_return_native():
+    assert isinstance(pd.Series([1, 2], dtype="int64").tolist()[0], int)
+    assert isinstance(pd.Series([1, 2], dtype="Int64").tolist()[0], int)
+    assert isinstance(pd.Series([1, 2], dtype="int64").to_dict()[0], int)
+    assert isinstance(pd.Series([1, 2], dtype="Int64").to_dict()[0], int)
+    assert isinstance(list(pd.Series([1, 2], dtype="int64").iteritems())[0][1], int)
+    assert isinstance(list(pd.Series([1, 2], dtype="Int64").iteritems())[0][1], int)
+    assert isinstance(list(iter(pd.Series([1, 2], dtype="int64")))[0], int)
+    assert isinstance(list(iter(pd.Series([1, 2], dtype="Int64")))[0], int)
```

Comment:
1) Please leave a comment with the issue number, so
```
def test_integer_Series_iter_return_native():
    # GH <issue number goes here>
```

2) Can this test be parametrised somewhat? See [here](https://docs.pytest.org/en/latest/parametrize.html), as well as several cases in the pandas tests (e.g. the one below this one) for examples. There's some examples in the [contributing guide](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#using-pytest) too